xcm/xcm-builder/src/location_conversion.rs

```diff
@@ -86,11 +86,22 @@ impl DescribeLocation for DescribeAccountKey20Terminal {
 
 pub type DescribeAccountIdTerminal = (DescribeAccountId32Terminal, DescribeAccountKey20Terminal);
 
+pub struct DescribeBodyTerminal;
+impl DescribeLocation for DescribeBodyTerminal {
+	fn describe_location(l: &MultiLocation) -> Option<Vec<u8>> {
+		match (l.parents, &l.interior) {
+			(0, X1(Plurality { id, part })) => Some((b"Body", id, part).encode()),
```

Comment:
You need to know where it came from?
```suggestion
			(0, X2(Parachain(para), Plurality { id, part })) => Some((b"Body", para, id, part).encode()),
```

Reply:
Not really. `DescribeFamily` handles this.